third_party/bigframes_vendored/pandas/io/gbq.py

```diff
@@ -71,6 +76,21 @@ def read_gbq(
             <BLANKLINE>
             [2 rows x 3 columns]
 
+        Reading data with `columns` and `filters` parameters:
+
+            >>> columns = ['pitcherFirstName', 'pitcherLastName', 'year', 'pitchSpeed']
+            >>> filters = [('year', '==', 2016), ('pitcherFirstName', 'in', ['John', 'Doe']), ('pitcherLastName', 'in', ['Gant'])]
+            >>> df = bpd.read_gbq(
+            ...             "bigquery-public-data.baseball.games_wide",
+            ...             columns=columns,
```

Comment:
In a future PR, let's make `columns` an alias for `col_order`.

```suggestion
            ...             col_order=columns,
```

Reply:
Will do.